src/backend/dbus/org.libproxy.proxy-system.service.in

```diff
@@ -0,0 +1,4 @@
+[D-BUS Service]
```

Comment:
`sudo zypper`

Reply:
All users missing `XAUTHORITY` and `DISPLAY` where no session bus can be started. (D-Bus session bus requirement).

Reply:
And yes, in this case it will fallback to env/sysconfig.

Reply:
Keep in mind that the main reason for using D-Bus was to make sure the is no more a crash introduced by different libs. And your proposal would lead to either move PxManager into libproxy-dbus version which would kill the benefit or have separate modules for those. I'm not sure this is worth the pain.

Reply:
What distros still use sysconfig for proxy configuration? Maybe that should best be implemented via downstream patch?

I think it's pretty reasonable if support for environment variables is implemented in the main process. But I also don't agree that not running on the system bus means we have to give up the benefits of D-Bus. Messages buses are an optional feature of D-Bus. You can talk D-Bus over any socket.

Reply:
Talked with Dom about sysconfig: OpenSuSE for sure and we at VW are also making use of it.

Implementing env in main process? Then we can drop d-bus. Because then we also need a pac parser here and curl. Furthermore the whole architecture needs to be changed.

We talked about transferring env during libproxy init to dbus server today. This could be a solution but not very elegant i think.

Reply:
The environment variable problem goes away when you use a private connection since the server process will be spawned by the client process.